pyproject.toml

```diff
@@ -15,29 +15,23 @@ dependencies = [
     "NREL-PySAM==4.2.0",
     "Pillow",
     "Pyomo>=6.1.2",
-    "diskcache",
```

Comment:
The docs build is failing because `alt_dev` requires this package. Was this just an unused import, or is it actually used in the model?

Reply:
Yea, just noticed that as well. It appears to be used in `alt_dev`, which I imagine several of the other packages were also used in. I've reached out to @dguittet to see if this directory should be kept around or not. Looks like the last development was 2 years ago (we moved some resource files 5 months ago, but that was done to the whole repo).

Reply:
Confirmed with @qualand that the directory could be removed for the next release.